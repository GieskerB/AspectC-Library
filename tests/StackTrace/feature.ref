Stack Trace Test: Tests for stack trace functionality by throwing an exception caught and uncaught.
====================================================================================================
-Testing Stack Trace with Caught Exception----------------------------------------------------------
Exception caught: foo-err
-Testing Stack Trace with Uncaught Exception--------------------------------------------------------


Uncaught exception foo-err:
	at void foo() in (main.cc:8)
	at void foo1(int,int) in (main.cc:10)
	at void foo2() in (main.cc:12)
	at void foo3(std::basic_string<char,std::char_traits<char>,std::allocator<char>>) in (main.cc:14)
	at void foo4() in (main.cc:28)
	at int main() in (main.cc:16)

