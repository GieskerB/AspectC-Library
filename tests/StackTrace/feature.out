Stack Trace Test: Tests for stack trace functionality by throwing an exception caught and uncaught.
====================================================================================================
-Testing Stack Trace with Caught Exception----------------------------------------------------------
DSMB: 0
DSMB: 0
DSMB: 0
Exception caught: foo-err
-Testing Stack Trace with Uncaught Exception--------------------------------------------------------
DSMB: 0
DSMB: 0
DSMB: 0
DSMB: 0
DSMB: 0
DSMB: 1
aösdfjlasdjföint main()
PENIS: (danke Felix)aösdfjlasdjföint main()
